pysrc/utils.py

```diff
@@ -68,3 +69,17 @@ def is_string(obj):
         return isinstance(obj, basestring)
     else:
         return isinstance(obj, str)
+
+
+def remove_arbitrary_identifiers(package_name: str) -> str:
+    """Removes arbitrary identifiers from package name
+
+    :param str package_name: package name to be processed
+    :returns: package_name without arbitrary identifiers
+    :rtype: string
+
+    opentelemetry-distro[otlp] -> opentelemetry-distro
+    """
+    package_name = re.sub(r'\[[^]]*]$', '', package_name)
```

Comment:
Ahhh regex scares me!

Reply:
But looking at it looks like it's linear time 😁

Reply:
Yea, not a friend of them either 😄 I'm looking for square brackets that have anything inside of them at the end of each row and replace it with nothing basically.

`package[dep123]` -> `package`

Reply:
You can always put that in a comment above 😁